agent-chat/public/index.html

```
@@ -0,0 +1,607 @@
+        <div class="toolbar">
+          <div class="tb-group">
+            <button id="provider-btn" class="chip" type="button" aria-haspopup="listbox">
+              <span class="dot" id="provider-dot"></span>
+              <span id="provider-name">Claude Code</span>
+              <svg class="chev" viewBox="0 0 10 6" width="10" height="6"><path d="M1 1l4 4 4-4" fill="none" stroke="currentColor" stroke-width="1.4" stroke-linecap="round" stroke-linejoin="round"/></svg>
+            </button>
+            <button id="cwd-chip" class="chip" type="button" title="working directory">
+              <svg viewBox="0 0 14 12" width="13" height="11" fill="none" stroke="currentColor" stroke-width="1.2"><path d="M1 3.4c0-.7.5-1.3 1.2-1.3h2.5l1.2 1.4h5.7c.7 0 1.2.6 1.2 1.3v4.9c0 .7-.5 1.3-1.2 1.3H2.2C1.5 11 1 10.4 1 9.7z"/></svg>
+              <span id="cwd-label"></span>
+            </button>
+            <input id="cwd-input" class="cwd-edit" spellcheck="false" hidden>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)